src/utils/arrayUtils.ts

```diff
@@ -25,3 +25,17 @@ export async function callChunked<Item>(
     await processFn(arrayChunk)
   }
 }
+
+/**
+ * Return a copy of the given array without null or undefined values
+ */
+export function removeNullish<const T>(array: readonly (T | null | undefined)[]): T[] {
+  return array.filter((e) => e !== undefined && e !== null) as T[]
+}
+
+/**
+ * Return a copy of the given array without falsy values (eg: false, 0, '', null, undefined)
+ */
+export function removeFalsy<const T>(array: readonly (T | null | undefined)[]): T[] {
```

Comment:
```suggestion
export function removeFalsy<const T>(array: readonly (T | null | undefined | 0 | '' | false)[]): T[] {
```

This provides better type narrowing, because these types will be excluded from inferred type then (they are assumed not to be a subset of T anymore)

Reply:
Really nice catch! I did it with `null` and `undefined` but I didn't think about the others values, really nice catch and suggestion! thank you so much Igor 🙇